include/rocksdb/cache.h

```diff
@@ -137,6 +144,57 @@ class Cache {
   // likely to get evicted than low priority entries.
   enum class Priority { HIGH, LOW };
 
+  // A set of callbacks to allow objects in the volatile block cache to be
```

Comment:
A general suggestion : Maybe we do not need to explicitly say it is designed for persistent cache tier usage. It is also possible that applications use block cache + another cache tier (can be either persistent or other cache designs). And we have a flag to indicate the secondary cache tier is persistent or not (to avoid cache warmup to use its persistency).

Reply:
Yeah, maybe referred to as a tiered cache will make it more general. For the flag to indicate persistent or not, I'm thinking, if it is persistent cache, we may need to check the consistency or correctness of the data in the cache when initialize the DB, such that the data in the tiered cache can be trusted. But for now, maybe we do not need it?